src/components/Modal.js

```diff
@@ -86,7 +86,7 @@ const Modal = ({ onDismiss, title, titleExtras, children, width = 450, showCance
   ])
 }
 
-Modal.propTypes = {
+export const modalPropTypes = {
```

Comment:
exporting to assist in Modal.mock.ts mock correctness.

Note that moving this module to Typescript will remove the need for propTypes entirely.  But wasn't going to shove that into this PR.